extensions/REMO/template/en/default/bug/create/created-mozreps.html.tmpl

```diff
@@ -35,9 +35,9 @@
 </p>
 
 <ul>
-  <li>The mozillians who vouched you should comment on the application [% terms.bug %] endorsing your application. Please contact them to make sure they endorse you, this is needed for your application to be complete.</li>
-  <li>The application will be reviewed after you are endorsed and if it fits all requirements it will be placed in the mentorship queue.</li>
-  <li>Once a Rep mentor has a free slot, they will be assigned to do the final review.</li>
+  <li>The Mozillians who vouched you should comment on the application [% terms.bug %] endorsing your application. Please contact them to make sure they endorse you, this is needed for your application to be complete.</li>
```

Comment:
is this still used? When testing locally I never saw this one. Maybe my local configuration does not have all the connections needed?